CMakeLists.txt

```diff
@@ -13,6 +13,11 @@ if(DEFINED ENV{NANO_SEC})
   add_definitions(-DBOOST_DATE_TIME_POSIX_TIME_STD_CONFIG)
 endif()
 
+option(HYPERSCAN "For Hyperscan regex engine" OFF)
+if(HYPERSCAN)
```

Comment:
please review the master/CMake.
it handles the arrow library, whether it exists or not (package is installed/no-installed)